### DISCUSSION TITLE
"semantic-release version" not updating pyproject.toml or git tag

### DISCUSSION THREAD

**Discussion by bergsmith** [semantic-release/semantic-release#2482] [Q&A]
I'm using PSR (v7.29.2) with poetry and when running `semantic-release version` I receive the following output:

```
Creating new version
Current version: 0.1.0, Current release version: 0.2.0

```
However, the pyproject.toml file still has `version = "0.1.0"` and no new tags are generated. Per the docs, this should "Figure out the new version number, update and commit it, and create a tag.", but seemingly nothing is happening except the new version determination. No warnings or errors are observed using `-v WARNING` or `-v ERROR`, respectively.

Any suggestions would be greatly appreciated!

**Comment by CandiedCode** [comment]
@bergsmith   Are you asking about https://github.com/relekang/python-semantic-release vs this semantic-release?
